workflows/phaseimpute/main.nf

```diff
@@ -145,7 +146,15 @@ workflow PHASEIMPUTE {
         VCF_SITES_EXTRACT_BCFTOOLS(VCF_NORMALIZE_BCFTOOLS.out.vcf_tbi)
         ch_versions = ch_versions.mix(VCF_SITES_EXTRACT_BCFTOOLS.out.versions)
 
-        // If required, phase panel (currently not working, a test should be added)
+        // Generate posfile channels from extracted sites
+        ch_posfile_glimpse = VCF_SITES_EXTRACT_BCFTOOLS.out.ch_posfile
+        ch_posfile_stitch = VCF_SITES_EXTRACT_BCFTOOLS.out.panel_tsv_stitch
+
+        // Concatenate all sites into a single VCF (for GLIMPSE concordance)
+        CONCAT_PANEL(VCF_SITES_EXTRACT_BCFTOOLS.out.panel_sites)
```

Comment:
If `posfile` is given then for the validation step, the sites should be concatenate across chromosomes for it to work.
I would suggest to move the `CONCAT_PANEL` to the validation step directly.